tests/integration/e2e/SchedulePage.test.tsx

```diff
@@ -193,7 +193,7 @@ describe('End-to-end Schedule Page tests', function () {
         const parentMeetingInfo = `${parentCourseDay}, ${parentStartTime} to ${parentEndTime} in ${parentLocation}`;
         const childCourseInfo = `${testChildCourse.prefix} ${testChildCourse.number} on ${parentMeetingInfo}`;
         await renderResult.findByText(childCourseInfo);
-      });
+      }).timeout(120000);
```

Comment:
Unless I'm missing something, I don't think you need this here as [this change is already on `develop`](https://github.com/seas-computing/course-planner/blob/develop/tests/integration/e2e/SchedulePage.test.tsx#L196).

Reply:
Thanks Robert! This was my attempt at fixing the build issues I was seeing.